client/webpack.config.js

```diff
@@ -10,7 +10,8 @@ module.exports = {
   entry: {
     application: './src/AppBundle/Resources/assets/javascripts/main.js',
     'formatted-report': './src/AppBundle/Resources/assets/scss/formatted-report.scss',
-    fonts: './src/AppBundle/Resources/assets/scss/fonts.scss'
+    fonts: './src/AppBundle/Resources/assets/scss/fonts.scss',
+    overview: './src/AppBundle/Resources/assets/javascripts/pages/overview.js'
```

Comment:
Wondered why you wanted to split this into a new JS file rather than keep it in `main.js`? Seems a bit more complex to maintain/reuse given our pretty sparse usage of JavaScript.

Reply:
So this comes from working with Webpacker in Rails and Encore in Symfony. I prefer to only have the JS required for the page to be loaded rather than bringing everything in.

It's a fair point about our limited JS though...I'm not completely married to the idea so happy to change. If we do start to use increasing amounts of JS in Digideps then it could be worthwhile following this pattern to reduce the number of unrequired assets loaded.